comp/core/log/mock/go.mod

```diff
@@ -42,7 +45,7 @@ require (
 	github.com/mitchellh/mapstructure v1.1.2 // indirect
 	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826 // indirect
 	github.com/pelletier/go-toml v1.2.0 // indirect
-	github.com/spf13/afero v1.1.2 // indirect
+	github.com/spf13/afero v1.11.0 // indirect
```

Comment:
This and a few other packages had a large version jump. Is it expected? It may be better to separate version bumps in a separate PR if this is necessary.

Reply:
This is already the current version in the main `go.mod` file so I think there is no actual change when building the agent (often other `go.mod` files are not updated...)